commit b5a2927b633d2ad357febe13efc717b02f04a5ba
Author: Ingmar Vanhassel <ingmar.vanhassel@gmail.com>
Date:   Sat Jan 12 15:28:44 2008 +0100

    Make Nepomuk & Soprano optional dependencies.
    Move the macro_optional_find_package({nepomuk,soprano}) statements before checking the results.r
---
 kdebase/apps/dolphin/src/CMakeLists.txt |   10 +++++-----
 1 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/kdebase/apps/dolphin/src/CMakeLists.txt b/kdebase/apps/dolphin/src/CMakeLists.txt
index f9902dc..c379e95 100644
--- a/kdebase/apps/dolphin/src/CMakeLists.txt
+++ b/kdebase/apps/dolphin/src/CMakeLists.txt
@@ -1,3 +1,8 @@
+macro_optional_find_package(Nepomuk)
+macro_optional_find_package(Soprano)
+macro_bool_to_01(Nepomuk_FOUND HAVE_NEPOMUK)
+
+configure_file(config-nepomuk.h.cmake ${CMAKE_CURRENT_BINARY_DIR}/config-nepomuk.h )
 
 add_subdirectory( pics )
 add_subdirectory( tests )
@@ -40,11 +45,6 @@ kde4_add_kcfg_files(dolphinprivate_LIB_SRCS
 
 kde4_add_library(dolphinprivate SHARED ${dolphinprivate_LIB_SRCS})
 
-find_package(Nepomuk)
-find_package(Soprano)
-macro_bool_to_01(Nepomuk_FOUND HAVE_NEPOMUK)
-
-configure_file(config-nepomuk.h.cmake ${CMAKE_CURRENT_BINARY_DIR}/config-nepomuk.h )
 
 target_link_libraries(dolphinprivate ${KDE4_KFILE_LIBS} konq ${BLITZ_LIBRARIES})
 if (Nepomuk_FOUND)
